src/Jade/Engine/PugJsEngine.php

```diff
@@ -54,7 +54,15 @@ protected function getPugJsOptions(&$input, &$filename, &$vars, &$pug)
             'out' => $workDirectory,
         );
         if (!empty($vars)) {
-            $options['obj'] = json_encode($vars, JSON_UNESCAPED_SLASHES);
+            $locals = json_encode($vars, JSON_UNESCAPED_SLASHES);
+
+            $options['obj'] = $locals;
+
+            if ($this->options['localsJsonFile']) {
+                $optionsFile = $workDirectory . '/options-' . hash('md5', time()) . '.json';
```

Comment:
`hash` was replaced with `mt_rand` like in https://github.com/pug-php/pug/blob/master/src/Jade/Engine/PugJsEngine.php#L46